build/lib/slugify.js

```diff
@@ -1,6 +1,6 @@
 const slugs = require('slugify');
 
 module.exports = exports = function slugify (s) {
-  const result = slugs(s, { remove: /[*+~.,()'"!?:@/\\]/g }).toLowerCase();
+  const result = s.trim().replace(/[^\p{L}\p{N}]+/ug, '-').replace(/^-+/, '').replace(/-+$/, '').toLowerCase();
```

Comment:
Have you checked if this changes any of the existing urls? I don't want to break any links.